HPXMLtoOpenStudio/resources/weather.rb

```diff
@@ -310,14 +311,57 @@ def calc_ground_temperatures
     phi = Math::atan(z)
     bo = (data.MonthlyAvgDrybulbs.max - data.MonthlyAvgDrybulbs.min) * 0.5
 
-    # The regression coefficients are from L. Xing's simplified design model ground temperatures (Appendix A-3) and the nearest TMY3 station's annual dry bulb.
-    data.GroundAnnualTemp = UnitConversions.convert(0.91 * UnitConversions.convert(data.AnnualAvgDrybulb, 'F', 'C') + 1.82, 'C', 'F')
-
-    data.GroundMonthlyTemps = []
+    data.ShallowGroundMonthlyTemps = []
     for i in 0..11
       theta = amon[i] * 24.0
-      data.GroundMonthlyTemps << data.GroundAnnualTemp - bo * Math::cos(2.0 * Math::PI / p * theta - po - phi) * gm
+      data.ShallowGroundMonthlyTemps << UnitConversions.convert(data.AnnualAvgDrybulb - bo * Math::cos(2.0 * Math::PI / p * theta - po - phi) * gm + 460.0, 'R', 'F')
+    end
+    data.ShallowGroundAnnualTemp = data.AnnualAvgDrybulb
+
+    if header.Latitude < 0
+      # Southern hemisphere
+      data.ShallowGroundMonthlyTemps.rotate!(6)
+    end
+  end
+
+  def calc_deep_ground_temperatures
+    # Return deep annual ground temperature.
```

Comment:
Since there is a small runtime penalty for retrieving this, can we skip this if there's no ground-source heat pumps to be modeled? That will also ensure that we aren't using these temperatures inappropriately for other things.

Reply:
And skip the `model.getSiteGroundTemperatureDeep` portion of `Location.apply_ground_temps` as well?

Reply:
Good point. Yes, skip that too.